frontend/src/app/about/page.tsx

```diff
@@ -159,20 +159,38 @@ const About = () => {
         </SecondaryCard>
 
         <SecondaryCard icon={faMapSigns} title={<AnchorTitle title="Roadmap" />}>
-          <ul>
-            {roadmap.map((item) => (
-              <li key={item.title} className="mb-4 flex flex-row items-center gap-2 pl-4 md:pl-6">
-                <div className="h-2 w-2 flex-shrink-0 rounded-full bg-gray-600 dark:bg-gray-300"></div>
-                <Link
-                  href={item.issueLink}
-                  target="_blank"
-                  className="text-gray-600 hover:underline dark:text-gray-300"
-                >
-                  {item.title}
-                </Link>
-              </li>
+          <div className="grid gap-4">
+            {projectMetadata.recentMilestones.map((milestone, index) => (
+              <div
+                key={index}
+                className={`flex items-center gap-4 overflow-hidden rounded-lg bg-gray-200 p-6 dark:bg-gray-700 ${
+                  index % 2 === 0 ? '' : 'ml-8'
+                }`}
+              >
+                <div className="flex-1">
+                  <Link
+                    href={milestone.url}
+                    target="_blank"
+                    className="inline-block hover:underline"
+                  >
+                    <h3 className="mb-2 text-xl font-semibold text-blue-400">{milestone.title}</h3>
+                  </Link>
+                  <p className="text-gray-600 dark:text-gray-300">{milestone.body}</p>
+                  <div className="mt-2 flex items-center gap-4">
```

Comment:
Also, can we remove progress block for now? I like hte idea, but with how milestone completion is claculated - this might be confusing. It might decrease significantly after showing 95% for example. So I think it would be better to now show any specific completion numbers.